lib/forceTty.js

```diff
@@ -0,0 +1,22 @@
+import { constants } from 'node:fs'
+import fs from 'node:fs/promises'
+import { WriteStream } from 'node:tty'
+
+export const forceTty = async () => {
+  if (process.stdout.isTTY) return null
+
+  try {
+    const handle = await fs.open('/dev/tty', constants.O_WRONLY + constants.O_NOCTTY)
```

Comment:
It'll fail on Windows always?

Reply:
This might be helpful on Windows: https://github.com/indutny/reopen-tty/blob/e92a1118497b8d5df3fcfd60577b9bdd4e9d563b/index.js